contracts/modules/cover/Cover.sol

```diff
@@ -0,0 +1,364 @@
+
+import "@openzeppelin/contracts-v4/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts-v4/token/ERC20/IERC20.sol";
+import "../../interfaces/ICover.sol";
+import "../../interfaces/IStakingPool.sol";
+import "../../interfaces/IPool.sol";
+import "../../abstract/MasterAwareV2.sol";
+import "../../interfaces/IMemberRoles.sol";
+import "hardhat/console.sol";
+
+
+contract Cover is ICover, ERC721, MasterAwareV2 {
+
+  Cover[] public override covers;
+  mapping(uint => StakingPool[]) stakingPoolsForCover;
+
+  Product[] public products;
+  mapping(uint => uint) capacityFactors;
```

Comment:
Remove this since we  already have it in Product struct.

Reply:
actually this can just be a global param `capacityFactor` initialized to 2x